src/services/codefixes/importFixes.ts

```diff
@@ -0,0 +1,393 @@
+                    }
+                    return getCodeActionForNamespaceImport(declaration.name.getText());
+
+                    function getTextChangeForImportList(importList: NamedImports): TextChange {
+                        if (importList.elements.length === 0) {
+                            const start = importList.getStart();
+                            return {
+                                newText: `{ ${name} }`,
+                                span: { start, length: importList.getEnd() - start }
+                            };
+                        }
+
+                        // Insert after the last element
+                        const insertPoint = importList.elements[importList.elements.length - 1].getEnd();
+
+                        // If the import list has one import per line, preserve that. Otherwise, insert on same line as last element
+                        let oneImportPerLine: boolean;
+                        if (importList.elements.length === 1) {
+                            /**
+                             * If there is only one symbol being imported, still check to see if it's set up for multi-line imports like this:
+                             *     import {
+                             *         foo
+                             *     } from "./module";
+                             */
+                            const startLine = getLineOfLocalPosition(sourceFile, importList.getStart());
+                            const endLine = getLineOfLocalPosition(sourceFile, importList.getEnd());
+                            oneImportPerLine = endLine - startLine >= 2;
+                        }
+                        else {
+                            const startLine = getLineOfLocalPosition(sourceFile, importList.elements[0].getStart());
+                            const endLine = getLineOfLocalPosition(sourceFile, insertPoint);
+                            oneImportPerLine = endLine - startLine >= importList.elements.length - 1;
+                        }
+
+                        return {
+                            newText: `,${oneImportPerLine ? context.newLineCharacter : ""}${name}`,
+                            span: { start: insertPoint, length: 0 }
+                        };
+                    }
+
+                    function getCodeActionForNamespaceImport(namespacePrefix: string): CodeAction {
+                        /**
+                         * Cases:
+                         *     import * as ns from "mod"
+                         *     import default, * as ns from "mod"
+                         *     import ns = require("mod")
+                         *
+                         * Because there is no import list, we alter the reference to include the
+                         * namespace instead of altering the import declaration. For example, "foo" would
+                         * become "ns.foo"

```

Comment:
I ended up providing two code actions to the user, and he/she can do either one. I think C# has the same behavior.